nautobot/core/apps/__init__.py

```diff
@@ -496,8 +510,10 @@ def fixed_fields(self):
 
     permissions = []
     buttons = []
+    args = []
+    kwargs = {}
 
-    def __init__(self, link, name, permissions=None, buttons=(), weight=1000):
+    def __init__(self, link, name, args=[], kwargs={}, permissions=None, buttons=(), weight=1000):
```

Comment:
Don't default parameter values to mutable objects. These should default to `None` and be set to `[]` or `{}` in the method body if needed.